app/templates/show-project.hbs

```diff
@@ -48,7 +51,12 @@
           </p>
           {{#if model.dcp_ulurp_nonulurp}}
             <p class="cell small-6 text-right">
-              <span class="label dark-gray">{{model.dcp_ulurp_nonulurp}}<sup>{{icon-tooltip tip='Uniform Land Use Review Procedure (ULURP) is a procedure whereby applications affecting the land use of the city are publicly reviewed within mandated time frames. A project is designated ULURP if a minimum of one action (e.g. a Zoning Map Amendment) within the application process is designated ULURP. Key participants in the process are the Department of City Planning (DCP), the City Planning Commission (CPC), Community Boards, the Borough Presidents, the Borough Boards, the City Council and the Mayor.'}}</sup></span>
+              <span class="label dark-gray">
+                {{model.dcp_ulurp_nonulurp}}
```

Comment:
Breaking this to multiple lines will add extra whitespace, which adds space between the info icon and the text it's explaining. Use tildes (~) in the handlebars to remove the whitespace.